loops-and-arrays.js

```diff
@@ -1,44 +1,74 @@
 // ==========================================
-// Opdracht 1
-// Schrijf een script dat iedere naam in bovenstaande array vervangt door een koosnaampje, door er -"je" achter te plakken.
-// Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 namen zou bevatten!
+// Opdracht 1. Maak een for-loop die 3 keer "Hoera!" in de terminal logt
+// Verwachte uitkomsten:
+// Hoera!
+// Hoera!
+// Hoera!
 // ==========================================
 
-const names = ["Henk", "Piet", "Fred", "Joop"];
-
-// Verwachtte uitkomsten:
-// Vóór het script zie je de originele waardes:
-// console.log(names) geeft: ["Henk", "Piet", "Fred", "Joop"]
-// Na jouw script zie je de aangepaste waardes:
-// console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
+for (let i = 0 ; i < 3; i++ ) {
+    console.log("Hoera!");
+}
 
 
 // ==========================================
-// Opdracht 2
-// Schrijf een script dat ieder getal in bovenstaande array met 3 vermenigvuldigd.
-// Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
+// Opdracht 2. Maak een for-loop die 4 keer het woord "loop..." logt, en bij de vijfde loop het woord "klaar!"
```

Comment:
Het lijkt erop dat de opdracht loops and arrays niet goed is gepushed naar de repository, ik heb enkel de code voor de loops opdracht. Deze kan ik helaas niet reviewen.
Over het algemeen ziet alle code er erg netjes en goed leesbaar uit.